components/builder-originsrv/src/migrations/origin_channels.rs

```diff
@@ -130,9 +130,7 @@ pub fn migrate(migrator: &mut Migrator) -> Result<()> {
                           WHERE o.name = op_origin
                           AND oc.name = op_channel
                           AND op.ident LIKE (op_ident  || '%')
-                          AND op.target = op_target
-                          ORDER BY op.ident DESC
-                          LIMIT 1;
+                          AND op.target = op_target;
```

Comment:
I think we should make it a v2 in a new migration. yeah? Seems like manually running ad hoc alter queries is not something we want to do outside of an emergency (if then at all).